doc/mgmt/Management Framework.md

```diff
@@ -1459,6 +1847,8 @@ CVL uses SONiC YANG models written based on ABNF schema along with various const
 
 SONiC YANG can be used as Northbound YANG for management interface by adding other data definitions such as state data (read only data), RPC or Notification as needed.Such YANG models are called SONiC NBI YANG models. Since CVL validates configuration data only, these data definition statements are ignored by CVL. During build time CVL YANG is actually generated from SONiC NBI YANG models with the help of CVL specific pyang plugin. 
 
+CVL supports multiple user-defined Redis database instances based on the [Multi DB instance HLD](https://github.com/Azure/SONiC/blob/master/doc/database/multi_database_instances.md) specification. During CVL initialization time, the DB configuration file is read from the predefined location and details are stored in internal data structure. CVL implements internal API for connecting to a Redis DB instance using the details stored in its internal data structure.
```

Comment:
what are the use case for multi redis db ? how does it matter w.r.t management framework?

Reply:
Mgmt framework code has its own DB access APIs (there is no swss DB APIs in Go). It should honor the new database config file as described in multi db HLD.